performance-tests/bench_2/common/util.h

```diff
@@ -14,21 +14,22 @@
 
 namespace {
```

Comment:
This was only in an anonymous namespace to protect it from symbol collisions. Now that it's being upgraded to a proper utility, let's move it into the Bench namespace below (along with ZERO_TIME) and put the implementation for one_at_a_time_hash in the .cpp and not the header.